src/components/map/layers/edit/NewRoutePathLayer.tsx

```diff
@@ -56,14 +56,20 @@ class RoutePathLayer extends Component<IRoutePathLayerProps> {
     }
 
     private renderLink = (routePathLink: IRoutePathLink) => {
+        const onClick = this.props.toolbarStore!.selectedTool ?
+            this.props.toolbarStore!.selectedTool!.onRoutePathLinkClick ?
+                this.props.toolbarStore!.selectedTool!.onRoutePathLinkClick!(routePathLink.id)
+                : undefined
+            : undefined;
```

Comment:
Yeah, it can be simplified, thanks! The name is super generic, since the function is generic. The function might be whatever the tool wants it to be. Another name could be for example: onRoutePathClick()?

Reply:
This is true! I think the new name is better and good enough